Parser/StarmonMiniParse.m

```diff
@@ -314,7 +314,7 @@
     iContent = i + nColumnTime;
     
     switch header.param(iParam).column
-        case 'Temp(�C)'
+        case 'Temp0x280xB0C0x29' %degrees C
```

Comment:
The textscan performed earlier in the function has a wrong format. I'll submit the necessary change.